apps/src/code-studio/header.js

```diff
@@ -180,8 +183,8 @@ function shareProject() {
           icon={appOptions.skin.staticAvatar}
           shareUrl={shareUrl}
           isAbusive={dashboard.project.exceedsAbuseThreshold()}
-          isSignedIn={appOptions.isSignedIn}
-          isPublished={dashboard.project.isPublished()}
+          canPublish={canPublish}
+          isPublished={!!dashboard.project.isPublished()}
```

Comment:
You've got me here. I thought I saw warnings about `isPublished` being a missing required prop type from within the app (not storybook), but now I can't see how that would be possible.